src/main/java/net/openhft/chronicle/wire/WireMarshaller.java

```diff
@@ -461,6 +462,10 @@ public boolean isLeaf() {
             offset = unsafeObjectFieldOffset(field);
             key = field::getName;
             this.isLeaf = isLeaf;
+
+            if ((field.getModifiers() & Modifier.FINAL) != 0)
```

Comment:
Suggest: `Modifier.isFinal(field.getModifiers))`

Reply:
We seem to always use the form I've used, can we stick with it for consistency?